packages/support/src/settings/_variables.color.scss

```diff
@@ -1,145 +1,22 @@
-<div class="ds-l-container">
-  <div class="ds-l-row">
-    <% colors.forEach(color => { -%>
-      <article class="ds-l-col--6 ds-l-md-col--4 ds-l-lg-col--3 ds-u-padding-x--1 ds-u-padding-bottom--2 c-swatch">
-        <div class="c-swatch__preview ds-u-fill--<%= color %>"></div>
-        <strong class="js-swatch-hex">#</strong>
-        <code class="c-swatch__label">
-          $color-<%= color %>
-        </code>
-      </article>
-    <% }) -%>
-  </div>
-
-Style guide: style.color.secondary
-*/
-
-/*
-Background colors
-
-@hide-markup
-
-These colors are used largely for background blocks and large content areas. When alternating between tones, be sure to use enough contrast between adjacent colors.
-
-Markup:
-<% var colors = [
-  'background', 'background-inverse', 'gray-dark', 'gray', 'gray-light', 'gray-lighter', 'gray-lightest'
-]; -%>
-<div class="ds-l-container">
-  <div class="ds-l-row">
-    <% colors.forEach(color => { -%>
-      <article class="ds-l-col--6 ds-l-md-col--4 ds-l-lg-col--3 ds-u-padding-x--1 ds-u-padding-bottom--2 c-swatch">
-        <div class="c-swatch__preview ds-u-fill--<%= color %>"></div>
-        <strong class="js-swatch-hex">#</strong>
-        <code class="c-swatch__label">
-          $color-<%= color %>
-        </code>
-      </article>
-    <% }) -%>
-  </div>
-
-Style guide: style.color.background
-*/
-
-/*
-Status colors
-
-@hide-markup
-
-These colors are used largely to indicate the status of something. Be cautious using these colors to signify something that may contradict what most people would expect the colors to be used for.
-
-Markup:
-<% var colors = [
-  'success', 'success-light', 'success-lighter', 'success-lightest',
-  'warn', 'warn-light', 'warn-lighter', 'warn-lightest',
-  'error-darkest', 'error-dark', 'error', 'error-light', 'error-lighter', 'error-lightest',
-]; -%>
-<div class="ds-l-container">
-  <div class="ds-l-row">
-    <% colors.forEach(color => { -%>
-      <article class="ds-l-col--6 ds-l-md-col--4 ds-l-lg-col--3 ds-u-padding-x--1 ds-u-padding-bottom--2 c-swatch">
-        <div class="c-swatch__preview ds-u-fill--<%= color %>"></div>
-        <strong class="js-swatch-hex">#</strong>
-        <code class="c-swatch__label">
-          $color-<%= color %>
-        </code>
-      </article>
-    <% }) -%>
-  </div>
-
-Style guide: style.color.status
-*/
-
 // Generic
 // USWDS doesn't include a "lighter" variant for red, so we create one
 // in order to be consistent with the other "state" colors.
 $color-secondary-lighter: mix($color-secondary-light, $color-secondary-lightest)
   !default;
 
+// USE COLOR RED HERE
 $color-error: $color-secondary !default;
 $color-error-dark: $color-secondary-dark !default;
 $color-error-darkest: $color-secondary-darkest !default;
 $color-error-light: $color-secondary-light !default;
 $color-error-lighter: $color-secondary-lighter !default;
 $color-error-lightest: $color-secondary-lightest !default;
 
+$color-red: $color-secondary !default;
```

Comment:
I feel like this belongs in `_variables.scss` like this:
`$color-red: #e31c3d !default;`

That way we can define `$color-error` like this:
`$color-error: $color-red !default;`

This makes it consistent with how the other status color variables are defined, and then we can add a comment in `_variables.scss`  about deprecating `$color-secondary` in the future

Reply:
I thought about that and tried that originally however when building the site any changes that are made to `_variables.scss` which is in vendor>uswds>core are overwritten. Also in the `_variables.scss` file there is a note to not modify that file. 

Maybe something we need to look into is what are we using the USWDS for aside from that variables file? I'm wondering if we could have our own variables file and break this dependency.

Reply:
Ah I gotcha. Yeah I agree, I think it's probably worth breaking with USWDS. I think USWDS v2 is even more different than what we have defined in that `vendor/uswds/core` file. The colors they have on their is much more comprehensive, while we have something much more specific. If we file a ticket to break out variables away from USWDS I'm ok with this being here

Reply:
https://jira.cms.gov/browse/WNMGDS-330